src/modules/media/controllers/media.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Body, Controller, Get, Logger, Param, Post } from '@nestjs/common';
+
+import { MediaQueryDto } from '../dto/media.dto';
+import { MediaService } from '../services/media.service';
+
+@Controller('media')
+export class MediaController {
+	private logger: Logger = new Logger(MediaController.name, { timestamp: true });
+
+	constructor(private mediaService: MediaService) {}
+
+	@Post()
+	public async getMedia(@Body() query: MediaQueryDto): Promise<any> {
+		const media = await this.mediaService.findAll(query);
```

Comment:
maybe rename query to queryInfo or queryObj so we keep a differentiation between:
* query the object containing filters and paging info
* and query the text a user inputs to search for